packages/flutter/lib/src/rendering/sliver_group.dart

```diff
@@ -114,12 +114,12 @@ class RenderSliverCrossAxisGroup extends RenderSliver with ContainerRenderObject
 
   @override
   void paint(PaintingContext context, Offset offset) {
-    RenderSliver? child = firstChild;
+    RenderSliver? child = lastChild;
```

Comment:
Is this change in the cross axis group intentional?

Reply:
I don't think so, I must've changed it thinking it was the `paint` method for main axis group.